src/common/copilot/IntelligenceApiService.ts

```diff
@@ -72,7 +72,7 @@ export async function sendApiRequest(userPrompt: string, activeFileParams: IActi
         const jsonResponse = await response.json();
 
         if (jsonResponse.operationStatus === 'Success') {
-          sendTelemetryEvent(telemetry, { eventName: CopilotResponseSuccessEvent, copilotSessionId: sessionID, durationInMills: responseTime });
+          sendTelemetryEvent(telemetry, { eventName: CopilotResponseSuccessEvent, copilotSessionId: sessionID, durationInMills: responseTime, orgId: orgID });
```

Comment:
Where is this OrgId going? Is it properly marked as PII data?

Reply:
Organization Identifiable Information (OII) - still needs to be properly handled

Reply:
Even today orgId is being logged by **AuthProfile** in customDimension. 

<img width="749" alt="image" src="https://github.com/microsoft/powerplatform-vscode/assets/97606074/e600e33f-b56a-4054-add2-0081e7b2aa56">